.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,36 @@
+name: "CodeQL"
+
+on:
+ push:
+   branches: [ master ]
+ pull_request:
+   branches: [ master ]
+ schedule:
+   - cron: '30 2 * * 0'
```

Comment:
Not sure about a scheduled run: who gets notified and who would fix these up?

Reply:
This was the default when I clicked "add code scanning" in the UI. As below with the `matrix`. I'll remove them, you're right, a bit redundant.